core/src/main/java/org/pentaho/platform/web/http/api/resources/JobScheduleRequest.java

```diff
@@ -166,15 +166,16 @@ public void setSimpleJobTrigger( SimpleJobTrigger jobTrigger ) {
     this.simpleJobTrigger = jobTrigger;
   }
 
-  public List getJobParameters() {
-    return (ArrayList<IJobScheduleParam>)(ArrayList<?>) jobParameters;
+  @XmlElement( type=JobScheduleParam.class )
```

Comment:
The current implementation of Jaxb was throwing an error about not handling interfaces + multiple places to set the unmarshalled objects.` @XmlElement` annotation handles both issues.

Previous error before:
```java
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: com.sun.xml.bind.v2.runtime.IllegalAnnotationsException: 2 counts of IllegalAnnotationExceptions
org.pentaho.platform.api.scheduler2.IJobScheduleParam is an interface, and JAXB can't handle interfaces.
	this problem is related to the following location:
		at org.pentaho.platform.api.scheduler2.IJobScheduleParam
		at public java.util.List org.pentaho.platform.web.http.api.resources.JobScheduleRequest.getJobParameters()
		at org.pentaho.platform.web.http.api.resources.JobScheduleRequest
Class has two properties of the same name "jobParameters"
	this problem is related to the following location:
		at public java.util.List org.pentaho.platform.web.http.api.resources.JobScheduleRequest.getJobParameters()
		at org.pentaho.platform.web.http.api.resources.JobScheduleRequest
	this problem is related to the following location:
		at java.util.List org.pentaho.platform.web.http.api.resources.JobScheduleRequest.jobParameters
		at org.pentaho.platform.web.http.api.resources.JobScheduleRequest
```